src/Symfony/Component/Yaml/README.md

```diff
@@ -1,21 +1,33 @@
 Yaml Component
 ==============
 
-YAML implements most of the YAML 1.2 specification.
+[![Package version](https://img.shields.io/packagist/v/symfony/yaml.svg)](https://packagist.org/packages/symfony/yaml)
 
-```php
-use Symfony\Component\Yaml\Yaml;
+The Yaml Component implements most of the
+[YAML 1.2 specification](http://yaml.org/spec/1.2/spec.html).
 
-$array = Yaml::parse(file_get_contents(filename));
+[See the Documentation](http://symfony.com/doc/current/components/yaml).
 
-print Yaml::dump($array);
+Contributing
```

Comment:
I would keep such a section, but instead, its content should tell people to use the main repository when contributing (making it less likely to have to move PRs to the main repo, which breaks the development experience as people cannot modify the moved PR after that as it is owned by a core dev)